include/audio/codec.h

```diff
@@ -84,6 +85,16 @@ typedef enum {
 	AUDIO_CHANNEL_ALL,
 } audio_channel_t;
 
+/*
+ * DAI Route types
+ */
+typedef enum {
+	AUDIO_ROUTE_BYPASS,
+	AUDIO_ROUTE_PALYBACK,
+	AUDIO_ROUTE_PACKBACK_CAPTURE,
```

Comment:
```suggestion
	AUDIO_ROUTE_PLAYBACK_CAPTURE,
```
I assume 'PLAY' as well?

Reply:
yes, let me udpate